src/schema/objects/columns.yaml

```
@@ -592,7 +596,8 @@ strain:
     For species different from `homo sapiens`, string value indicating
     the strain of the species, for example: `C57BL/6J`.
   definition: {
-    "Description": "name of the strain of the species"
+    "Description": "name of the strain of the species",
```

```suggestion
    "Description": "Name of the strain of the species.",
```
Standardizing the punctuation among some of the changed descriptions (not holistic, just as I see them changed here)